tests/integration/explorer_blueprint/test_explorer.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+from pathlib import Path
+
+import pytest
+from fastapi import FastAPI
+from starlette.testclient import TestClient
+
+from antarest.study.model import NonStudyFolder, WorkspaceMetadata
+
+BAD_REQUEST_STATUS_CODE = 400
+# Status code for directory listing with invalid parameters
+INVALID_PARAMS_STATUS_CODE = 422
+
+
+@pytest.fixture
+def study_tree(tmp_path: Path) -> Path:
+    """
+    Create this hierarchy
+
+    tmp_path
+    └── ext_workspace
+        └── folder
+            ├── studyC
+            │   └── study.antares
+            ├── trash
+            │   └── trash
+            ├── another_folder
+            │   ├── AW_NO_SCAN
+            │   └── study.antares
+    """
+    ext_workspace = tmp_path / "ext_workspace"
+    c = ext_workspace / "folder/studyC"
+    c.mkdir(parents=True)
+    (c / "study.antares").touch()
+
+    d = ext_workspace / "folder/trash"
+    d.mkdir(parents=True)
+    (d / "trash").touch()
+
+    f = ext_workspace / "folder/another_folder"
+    f.mkdir(parents=True)
+    (f / "AW_NO_SCAN").touch()
+    (f / "study.antares").touch()
+
+    return tmp_path
+
+
+def test_explorer(client: TestClient, admin_access_token: str, study_tree: Path):
+    workspace = "ext"
```

Comment:
Could you plz add a little line of doc explaining what you told me by Teams about the TestClient ?